reactor-netty-http/src/main/java/reactor/netty/http/client/HttpConnectionProvider.java

```diff
@@ -76,7 +76,7 @@ public Map<SocketAddress, Integer> maxConnectionsPerHost() {
 	final AtomicReference<ConnectionProvider> h2ConnectionProvider = new AtomicReference<>();
 
 	HttpConnectionProvider() {
-		this(null);
+		this(ConnectionProvider.create("default", 1));
```

Comment:
I made new ConnectionProvider. 
Because If `http1ConnectionProvider` and `h2ConnectionProvider.get()` values are all null, 
It's impossible to get original ConnectionProvider